utils/spacewalk-common-channels.ini

```diff
@@ -1207,17 +1208,82 @@ repo_url = https://download.opensuse.org/update/leap-micro/5.3/sle/
 name     = Uyuni Client Tools for %(base_channel_name)s
 archs    = x86_64, aarch64
 base_channels = opensuse_micro5_3-%(arch)s
+=======
+[opensuse_leap15_5]
+checksum = sha256
+archs    = x86_64, aarch64
+name     = openSUSE Leap 15.5 (%(arch)s)
+gpgkey_url = http://download.opensuse.org/distribution/leap/15.5/repo/oss/repodata/repomd.xml.key
+gpgkey_id = 3DBDC284
+gpgkey_fingerprint = 22C0 7BA5 3417 8CD0 2EFE  22AA B88B 2FD4 3DBD C284
+repo_url = http://download.opensuse.org/distribution/leap/15.5/repo/oss/
+dist_map_release = 15.5
+
+[opensuse_leap15_5-non-oss]
+label    = %(base_channel)s-non-oss
+name     = openSUSE 15.5 non oss (%(arch)s)
+archs    = x86_64, aarch64
+checksum = sha256
+base_channels = opensuse_leap15_5-%(arch)s
+repo_url = http://download.opensuse.org/distribution/leap/15.5/repo/non-oss/
+
+[opensuse_leap15_5-updates]
+label    = %(base_channel)s-updates
+name     = openSUSE Leap 15.5 Updates (%(arch)s)
+archs    = x86_64, aarch64
+checksum = sha256
+base_channels = opensuse_leap15_5-%(arch)s
+repo_url = http://download.opensuse.org/update/leap/15.5/oss/
+
+[opensuse_leap15_5-non-oss-updates]
+label    = %(base_channel)s-non-oss-updates
+name     = openSUSE Leap 15.5 non oss Updates (%(arch)s)
+archs    = x86_64, aarch64
+checksum = sha256
+base_channels = opensuse_leap15_5-%(arch)s
+repo_url = http://download.opensuse.org/update/leap/15.5/non-oss/
+
+[opensuse_leap15_5-sle-updates]
+label    = %(base_channel)s-sle-updates
+name     = Update repository with updates from SUSE Linux Enterprise 15 (%(arch)s)
+archs    = x86_64, aarch64
+checksum = sha256
+base_channels = opensuse_leap15_5-%(arch)s
+repo_url = http://download.opensuse.org/update/leap/15.5/sle/
+
+[opensuse_leap15_5-backports-updates]
+label    = %(base_channel)s-backports-updates
+name     = Update repository of openSUSE Backports (%(arch)s)
+archs    = x86_64, aarch64
+checksum = sha256
+base_channels = opensuse_leap15_5-%(arch)s
+repo_url = http://download.opensuse.org/update/leap/15.5/backports/
+
+# This is expected. openSUSE Leap 15.0 client tools are valid for all openSUSE Leap 15.X releases
+[opensuse_leap15_5-uyuni-client]
+name     = Uyuni Client Tools for %(base_channel_name)s
+archs    = x86_64, aarch64
+base_channels = opensuse_leap15_5-%(arch)s
+>>>>>>> a5d671f3388 (Add openSUSE Leap 15.5 repositories to spacewalk-common-channels)
```

Comment:
Same here. It seems the whole conflict was not solved and you just used `git add`